packages/elements/src/pill/index.ts

```diff
@@ -95,7 +100,37 @@ export class Pill extends ControlElement {
     this.addEventListener('tapstart', this.onStartPress);
     this.addEventListener('tapend', this.onEndPress);
     this.addEventListener('mouseleave', this.onEndPress);
+    this.addEventListener('keydown', this.onKeyDown);
+
+  }
+
+  /**
+   * Updates the element
+   * @param changedProperties Properties that has changed
+   * @returns {void}
+   */
+  protected update (changedProperties: PropertyValues): void {
+    if (changedProperties.has('toggles') || changedProperties.has('active')) {
+      if (this.toggles) {
+        this.setAttribute('aria-pressed', String(this.active));
+      }
+      else {
+        this.removeAttribute('aria-pressed');
+      }
+    }
+
+    super.update(changedProperties);
+  }
 
+  /**
+   * Handles key down event
+   * @param event Key down event object
+   * @returns {void}
+   */
+  private onKeyDown (event: KeyboardEvent): void {
+    if (event.key === 'Delete' && (this.clears && !this.readonly)) {
```

Comment:
Did you check on IE11 ?

Reply:
For IE 11, I need to check for the `Del` key.
```suggestion
    if ((event.key === 'Delete' || event.key === 'Del') && (this.clears && !this.readonly)) {
```